lib/broadside/deploy.rb

```diff
@@ -4,9 +4,11 @@ class Deploy
 
     attr_reader :target
 
+    delegate :tag, to: :target
+
     def initialize(options = {})
       @target = Broadside.config.get_target_by_name!(options[:target])
-      @tag = options[:tag] || @target.tag
+      @target.tag = options[:tag] if options[:tag]
```

Comment:
@apurvis that links to the general PR? 

if you are bootstrapping a new service, that assertion checks if the _only_ the target's tag is set, but without changing either that line--or this one--it is never, under any circumstances, set. only the `Deploy` object has the correct `tag`. *nothing* appears to be using the `Deploy` object's tag, AFAICT.